importmagic/index.py

```diff
@@ -420,5 +442,4 @@ def visit_If(self, node):
 if __name__ == '__main__':
     # print ast.dump(ast.parse(open('pyautoimp.py').read(), 'pyautoimp.py'))
     tree = SymbolIndex()
-    tree.build_index(sys.path)
-    tree.serialize(sys.stdout)
+    tree.get_or_create_index()
```

Comment:
Can you revert this? Or add eg. a `--stdout` flag to dump to stdout for debugging.